Formula/n/nss.rb

```diff
@@ -33,11 +34,12 @@ class Nss < Formula
   conflicts_with "resty", because: "both install `pp` binaries"
 
   def install
-    # Fails on arm64 macOS for some reason with:
-    #   aes-armv8.c:14:2: error: "Compiler option is invalid"
-    ENV.runtime_cpu_detection if OS.linux? || Hardware::CPU.intel?
+    ENV.runtime_cpu_detection
     ENV.deparallelize
     cd "nss"
+    # Set architecture flags for ARM-based Macs correctly.
+    inreplace "coreconf/Darwin.mk", "# Nothing set for arm currently.",
+      "CC += -arch aarch64\nCCC += -arch aarch64\noverride CPU_ARCH	= aarch64"
```

Comment:
Is this based on a patch that's already been merged on your Phabricator? If not, we'd prefer that this be fixed there first (or, at least, concurrently, with a reference to the upstream fix here in the `inreplace`).

Reply:
I think this is a change we should just make in-tree. Our infra does build/test using make, but interestingly not on macOS, which is probably something we should change as well.